examples/object_detection/ios/ObjectDetector/Helpers/ObjectOverlayHelper.swift

```diff
@@ -0,0 +1,123 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import UIKit
+
+import MediaPipeTasksVision
+
+// Helper Methods for calculating bounding boxes that are to be drawn.
+class ObjectOverlayHelper {
```

Comment:
IMO having less class is better for sample apps. Can you move this helper logic into OverlayView so that all logic necessary to display the object detection result are in the same class?

Reply:
The helper imports MediaPipeTasksVision. I did not want the view to import it. Just to separate any detection to overlay conversion logic that the view doesn't need to know.

But it is not a necessity. LMK if this needs to be done.

Reply:
`OverlayView` is meant to draw an object returned by `MediaPipeTasksVision` so I don't see having it to import `MediaPipeTasksVision` being a problem. Can you merge OverlayView and ObjectOverlayHelper?

Reply:
Fixed